docs/source/user-guide/mission-modeler-guide/configuration/examples.md

```diff
@@ -0,0 +1,36 @@
+## Examples
+
+`Configuration` can be a simple data class. For example:
+
+```java
+package gov.nasa.jpl.aerie.banananation;
+
+import java.nio.file.Path;
+
+import static gov.nasa.jpl.aerie.merlin.framework.annotations.Export.Template;
+
+public record Configuration(int initialPlantCount, String initialProducer, Path initialDataPath) {
+
+  public static final int DEFAULT_PLANT_COUNT = 200;
+  public static final String DEFAULT_PRODUCER = "Chiquita";
+  public static final Path DEFAULT_DATA_PATH = Path.of("/etc/os-release");
+
+  public static @Template Configuration defaultConfiguration() {
+    return new Configuration(DEFAULT_PLANT_COUNT, DEFAULT_PRODUCER, DEFAULT_DATA_PATH);
+  }
+}
+```
+
+See [`examples/`](https://github.com/NASA-AMMOS/aerie/tree/develop/examples) for a demonstration of each possible style of configuration definitions:
+- `examples/foo-missionmodel`: uses standard `@Parameter` configuration annotations.
```

Comment:
Minor request: Could these be linked to the Configuration class on develop for each of these examples. (IE, for banananation, it would link to [here](https://github.com/NASA-AMMOS/aerie/blob/develop/examples/banananation/src/main/java/gov/nasa/jpl/aerie/banananation/Configuration.java)? I just noticed you have to go several directories deep to find the `Configuration` file on each of these.